tests/lazy-init.html

```diff
@@ -93,19 +111,65 @@
     }
   }
 
+  class LazyParent extends HTMLElement {
```

Comment:
Let's make one class here since these are the same.

Also, we should probably not call these `Lazy` and `Eager` and instead just something like `Host` and `Client`.

Reply:
Done!